alexandria/settings/alexandria.py

```diff
@@ -184,17 +184,20 @@ def default(default_dev=env.NOTSET, default_prod=env.NOTSET):
 ALEXANDRIA_MANABI_DAV_URL_PATH = env.str(
     "ALEXANDRIA_MANABI_DAV_URL_PATH", default="/dav"
 )
-ALEXANDRIA_MANABI_DAV_SCHEME = env.str(
```

Comment:
Counts as breaking right?

Reply:
Technically, yes. However, as this setting was completely useless in the past (it automatically does the correct thing - no one would configure HTTP instead of HTTPS in production) I'd suggest not doing a breaking release for this.

If we say it must be a breaking change then I'd rather declare it as such instead of doing a fallback as you suggested above as the previous setting made absolutely no sense (we allow excel files but then tell the browser to open them in Word).

@open-dynaMIX @winged @czosel Any inputs from your side regarding that matter?

Reply:
@open-dynaMIX @winged @czosel If you don't have an opinion regarding this topic I'd appreciate a short comment so we can go ahead with this.

Reply:
Given that the old setting didn't work, I don't think this needs to considered as breaking. Green light from my side